instance/static/html/instance/index.html

```diff
@@ -63,7 +68,13 @@
                                                ng-click="provision(selected.instance)">
               Reprovision
             </button>
-            <img src="/static/img/running.png" tooltip-placement="bottom" tooltip="Provisioning in progress..." alt="Provisioning in progress..." ng-switch-default />
+            <button class="button reprovision" ng-switch-when="provisioning_failure"
+                                               ng-click="provision(selected.instance)">
+              Reprovision
+            </button>
+            <i class="fa fa-2x fa-ellipsis-h" tooltip-placement="right" tooltip="Provisioning in progress" ng-switch-default></i>
+            <i class="fa fa-2x fa-times" tooltip-placement="right" tooltip="Provisioning failure" ng-switch-when="provisioning_failure"></i>
+            <i class="fa fa-2x fa-check" tooltip-placement="right" tooltip="Ready" ng-switch-when="ready"></i>
```

Comment:
@chromano I meant that the choice of status that get chosen for each (`provisioning_failure`, `ready`, etc.) is repeated in several places, which would require to update them all every time we add a status, and could thus be factorized in a controller method. But if we go with https://github.com/open-craft/opencraft/pull/13/files#r41096377 this would already be done on the server side, and could make it unecessary - it would guarantee that there would only be one value for each to test.